episodes/02-image-basics.md

```diff
@@ -122,7 +122,7 @@ import numpy as np             # form 4, load all of numpy into an object called
 
 ## Further Explanation
 
-In the example above, form 1 loads the entire `skimage` library into the
+In the example above, form 1 loads the entire scikit-image library into the
```

Comment:
Here I would leave `skimage` because we are referring to code. We are actually importing Python modules, which must have all-lowercase names...